ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,438 @@
+whenever you change something on a device.
+It's all done automatically based on the configuration built and applied to the network devices by Ansible, everytime you execute the playbooks.
+
+Below you will find two examples of documentation automatically created by Ansible AVD:
+
+- [5 stage CLOS fabric across two Data Centers](../molecule/eos_designs-twodc-5stage-clos/documentation/fabric/TWODC_5STAGE_CLOS-documentation.md)
+- [A single leaf inside the fabric above](../molecule/eos_designs-twodc-5stage-clos/documentation/devices/DC1-POD1-L2LEAF1A.md)
+
+## How do I use Ansible AVD
+
+Below are a number of links to additional documentation about Ansible AVD and Ansible in general:
+- With [eos_design](../../roles/eos_cli_config_gen)
+- With [eos_cli_config_gen](../../roles/eos_cli_config_gen)
+- With [Ansible Tower](https://docs.ansible.com/ansible/2.3/tower.html)
+- With CI/CD
+- With [eos_config_deploy_cvp](../../roles/eos_config_deploy_cvp) / [eos_config_deploy_eapi](../../roles/eos_config_deploy_eapi)
+
+Ansible AVD comes with pre-built templates that you can either decide to re-use as-is or adapt to your liking.
+
+Once the templates reflect your desired network configuration, you deploy the configuration either directly to the Arista EOS devices or via Arista Cloudvision using its internal change management system, where the changes can be reviewed, approved and executed in accordance with existing company requirements.
+The deployment is executed typically using CLI commands or via the GUI available with Ansible Tower. 
+
+For example, running the CLI command: 
+```
+ansible-playbook playbooks/dc1-fabric-deploy-cvp.yml --tags build
+```
+...will execute the playbook `playbooks/dc1-fabric-deploy-cvp.yml` with the tag `build`. 
+
+Once the configuration has been deployed, an automated validation of the designed configuration versus the running configuration can be executed using the included `validation` playbook, to ensure that:
+
+- all links are operational and cabled as desired
+- BGP adjacencies are operational
+- and more...
+
+This quickly identifies any issues that could otherwise be overlooked and a redundant link could sit non-operational for months unless cumbersome manual testing is executed again per-device throughout the fabric.
+
+## Day 2 Operations
+The term “Day 2 Operations” describes most, if not all network configuration changes that occur after the initial day 1 configuration.
+
+The really time consuming day-to-day tasks such as:
+- adding/removing new leaf switches
+- adding/removing new VLANs
+- creating/modifying/deleting port profiles
+- configuring/modifying ports towards end systems
+
+no longer have to be performed on a device-by-device basis across the entire fabric.
+
+A configuration change can easily be applied to the Ansible YAML files and the scope for a given change can be specified, e.g. should the new VLAN 100 be added to all or a subset of leaf switches. 
```

Comment:
Consider adding a snipped from the avd-cloudvision-demo repo as an example like you did before *(group variables section)